deploy/roles/citus/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Add Citus rpm
+  get_url:
+    url: "{{ rpm_url }}"
+    dest: /tmp/rpm.sh
+    mode: "0755"
+  tags:
+    - citus
+
+- name: Execute rpm script
+  become: yes
+  become_method: sudo
+  shell: ./rpm.sh
+  args:
+    chdir: /tmp/
+  when: not custom_repo
+  tags:
+    - citus
+
+- name: Install Citus
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ citus_pkg }}"
```

Comment:
@rioh Since the Enterprise version has some additional packages, can you change `citus_pkg` to `citus_pkgs`, which is a list instead using `with_items`?  That way we can easily adjust.